src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -349,25 +349,29 @@ public void buttonCvrFilePathClicked() {
     }
   }
 
+  private boolean cvrSourceHasRequiredFields(CVRSource source) {
+    boolean hasRequiredFields = true;
+    if (source.getFilePath().isBlank()) {
+      hasRequiredFields = false;
+      Logger.log(Level.WARNING, "filePath is required!");
```

Comment:
here and below: why only WARNING instead of SEVERE?

Reply:
Convention in the GUI. It's not making anything crash or throwing any real errors, it just prevents the add.